src/test/unit/EigenPodUnit.t.sol

```diff
@@ -88,23 +101,231 @@ contract EigenPodUnitTests is EigenLayerUnitTestSetup {
         cheats.store(address(eigenPod), slot, value);
         _;
     }
+
+    modifier timewarp() {
+        uint curState = timeMachine.warpToLast();
+        _;
+        timeMachine.warpToPresent(curState);
+    }
+
+    function _seedPodWithETH(uint256 ethAmount) internal {
+        cheats.deal(address(this), ethAmount);
+        bool result;
+        bytes memory data;
+        (result, data) = address(eigenPod).call{value: ethAmount}("");
+    }
+
+    function _setPodHasNotRestaked(EigenPodUser staker) internal {
+        // Write hasRestaked as false. hasRestaked in slot 52
+        bytes32 slot = bytes32(uint256(52)); 
+        bytes32 value = bytes32(0); // 0 == false
+        cheats.store(address(staker.pod()), slot, value);
+    }
+
+    function _newEigenPodStaker(uint256 rand) internal returns (EigenPodUser, uint256) {
+        string memory stakerName;
+
+        EigenPodUser staker;
+
+        stakerName = string.concat("Staker", numStakers.toString());
+        staker = new EigenPodUser(stakerName);
+
+        uint256 amount = bound(rand, 1 ether, 640 ether);
+        cheats.deal(address(staker), amount);
+
+        numStakers++;
+        return (staker, amount);
+    }
```

Comment:
Probably a decent simplification here -- have this method return `(EigenPodUser, EigenPod)`. I noticed that you aren't using the `uint` return value anywhere, and in most places you call this, you then spend an extra line after getting the pod.